scripts/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#
+# Run swiftformat on git commit
+# see: https://github.com/nicklockwood/SwiftFormat#git-pre-commit-hook
+#
+# Install with:
+#
+# $ brew install swiftformat
+# $ ln -sf ../../scripts/pre-commit .git/hooks/pre-commit
```

Comment:
I guess for us it's from the home folder, no?
```suggestion
# $ ln -sf scripts/pre-commit .git/hooks/pre-commit
```

Reply:
I think it has to be relative to the location of `pre-commit`:

```sh
$ cd SwifterSwift
$ ln -sf scripts/pre-commit .git/hooks/pre-commit
$ .git/hooks/pre-commit
command not found: .git/hooks/pre-commit

$ cd SwifterSwift
$ ln -sf ../../scripts/pre-commit .git/hooks/pre-commit
$ .git/hooks/pre-commit
... runs properly ...
```

I find this counterintuitive as well and I messed it up the first time we tried this with our app 😄

Reply:
Ah I see, in the first instance you're creating a link that's trying to find `.git/hooks/scripts/pre-commit`, instead of `.git/hooks/../../scripts/pre-commit`. Makes sense now.